addons/website/static/src/snippets/s_faq_collapse/000.scss

```diff
@@ -33,3 +43,48 @@
         margin-bottom: 0;
     }
 }
+
+.s_faq_collapse_light {
+    .accordion .card {
+        border-left: 0px;
+        border-right: 0px;
+        .card-header {
+            background-color: transparent;
+        }
+    }
+}
```

Comment:
XXX.scss files need to contain only one rule about the snippet. So add those inside the `.s_faq_collapse` selector with:
```scss
.s_faq_collapse {
    // ...

    &.s_faq_collapse_light {
        // ...
    }
    // ...
}
```